django/librarian/static/librarian/scripts.js

```diff
@@ -10,11 +10,18 @@ let librarianModalCloseHandler = null;
 
 updateLibrarianModalOnClose = (library_id) => {
   const modalEl = document.getElementById('editLibrariesModal');
-  const chatId = modalEl.getAttribute('data-chat-id');
   modalEl.removeEventListener('hidden.bs.modal', librarianModalCloseHandler);
   librarianModalCloseHandler = event => {
+    // Stop polling on element id="libraryModalPoller"
+    // by replacing it with empty div
+    const poller = document.getElementById('libraryModalPoller');
+    const newPoller = document.createElement('div');
+    newPoller.id = 'libraryModalPoller';
+    poller.parentNode.replaceChild(newPoller, poller);
```

Comment:
When the librarian modal is closed, we want it to stop polling for the document status updates. This slows the page down unnecessary since user is not looking at the status changes anymore.